website/content/en/pre-docs/tasks/running-pods.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "Running pods"
+linkTitle: "Running pods"
+weight: 10
+---
+
+If your pods have no requirements for how or where to run, you can let Karpenter choose nodes from the full range of available cloud provider resources.
+However, by taking advantage of Karpenter's model of layered constraints, you can be sure that the precise type and amount of resources needed are available to your pods.
+Reasons for constraining where your pods run could include:
+
+* Saving money by running in ways that are more efficient
+* Needing to run in zones where dependent applications or storage are available
+* Requiring certain kinds of processors or other hardware
+* Wanting to use techniques like topology spread to help insure high availability
+
+Your Kubernetes cluster operator adds the first layer of constraints by creating one or more provisioners.
+The next layer comes from you adding specifications to your Kubernetes pod deployments.
+When you request constraints, those constraints must fall within the provisioner constraints or the pods will not deploy.
+For example, if the provisioner sets limits that allow only a particular zone to be used, and a pod asks for a different zone, deployment will fail.
```

Comment:
I found this section confusing. Potential concepts to clarify:
- explicitly identify which layer is dominant 
- explicitly introduce the "request" implied by "When you request..."
- include an example of the key/value used in the example. perhaps "In other words, the value of podspec.requestedzone must be contained in provisioner.enabledzones" (I don't know the actual keys).

Reply:
Ellis reworded this section. Thanks for the input.